nsqd/client_v2.go

```diff
@@ -473,6 +473,9 @@ func (c *clientV2) UpgradeTLS() error {
 	}
 	c.tlsConn = tlsConn
 
+	var zeroTime time.Time
+	tlsConn.SetDeadline(zeroTime)
```

Comment:
I think you can also do `tlsConn.SetDeadline(time.Time{})` (but what you have is probably more clear)